exporter/exporter.collect.go

```diff
@@ -26,15 +26,15 @@ func (e *Exporter) CollectPrometheusMetric(ch chan<- semp.PrometheusMetric) {
 			} else {
 				up = 0
 				err = errors.New("Software only scrape target: \"" + dataSource.Name + "\". Please check documentation for valid targets.")
-				_ = level.Error(e.logger).Log("Hardware only  scrape target: \"" + dataSource.Name + "\". Please check documentation for valid targets.")
+				_ = level.Error(e.logger).Log("Software only  scrape target: \"" + dataSource.Name + "\". Please check documentation for valid targets.")
```

Comment:
```suggestion
				_ = level.Error(e.logger).Log("Hardware only  scrape target: \"" + dataSource.Name + "\". Please check documentation for valid targets.")
```

Reply:
Health also is not valid for hardware, so I think the software only error is correct

Reply:
I guess so. Software broker dont know that, but i dont have an appliance to test against.

But `level.Error` and `errors.New` should say the same.

Reply:
That's what's confusing me.  In the old version they were different and the change was to make them both the same.  Is that not what you're seeing?